Definitions/Chord

```diff
@@ -1,18 +1,27 @@
 class Chord:
-    def __init__(self, definition):
+    def __init__(self, definition, inversion):
         self.symbol = definition.symbol
         self.degrees = definition.degrees
         self.next = definition.next
-        self.inversion = definition.inversion
+        self.inversion = inversion
         
     def hasScaleDegree(self, note):
         return note in self.degrees
-    
+
+    @classmethod
+    def chordFromSymbol(cls, symbol, inversion):
+        fileName = symbol + "_info.json"
+        file = open(fileName)
```

Comment:
This presents somewhat of a memory leak, as the opening connection is never closed. When opening files in modern versions of python should be done with the `with` keyword:
```
with open(filename, "r") as chord_json:
    definition = json.load(chord_json)
    return cls(definition, inversion)
```

The "r" opens it in read mode, and this whole block handles opening and closing connections.